packages/forms/src/UIForm/fields/__snapshots__/TextArea.test.js.snap

```diff
@@ -0,0 +1,73 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Text field should render disabled input 1`] = `
+<FieldTemplate
+  description="my text input hint"
+  errorMessage="My error message"
+  id="myForm"
+  isValid={true}
+  label="My input title"
+  labelAfter={true}
+>
+  <textarea
+    autoFocus={true}
+    className="form-control"
+    disabled={true}
+    id="myForm"
+    label="My input title"
+    onChange={[Function]}
+    placeholder="Type something here"
+    readOnly={undefined}
+    type="text"
```

Comment:
It's the type in jsfc merged schema that transform the string to text
But this will go after the `type` removal in the textarea